contracts/Zauction.sol

```diff
@@ -13,6 +13,7 @@ contract ZAuction is Initializable, OwnableUpgradeable {
   using ECDSA for bytes32;
   using SafeERC20 for IERC20;
 
+  // Default ERC20 Token
   IERC20 public token;
```

Comment:
Is this here so we can default to $ZERO token when multiple networks exist? Say 0://wilder & 0://wizards.

Reply:
Yes we want to use it as a default if no network token is specified, e.g. imagine the KittyDAO uses $KITTY as their ERC20 governance token but they change their mind later so the unset that networks token. Having the default allows bids and purchases or NFTs in that network to still be made.

Also, removing this would break upgradeability because it would modify the storage layout for the smart contract. 

[Upgrading smart contracts](https://docs.openzeppelin.com/learn/upgrading-smart-contracts)

Reply:
Cool. Thanks for the explanation